src/bin/hq.rs

```diff
@@ -123,6 +129,8 @@ enum WorkerCommand {
     List(WorkerListOpts),
     /// Hwdetect
     Hwdetect,
+    /// Display details about worker
+    Details(WorkerDetailsOpts),
```

Comment:
I would still prefer "hq worker info <xxx>" as it is more compatible with the rest of the terminology. However, if there is a good reason why call it "detail", lets discuss it.